tensorflow_probability/python/experimental/mcmc/particle_filter_test.py

```diff
@@ -734,6 +735,210 @@ def marginal_log_likelihood(level_scale, noise_scale):
     self.assertAllNotNone(grads)
     self.assertAllAssertsNested(self.assertNotAllZero, grads)
 
+  def test_smc_squared_rejuvenation_parameters(self):
+    def particle_dynamics(params, _, previous_state):
+      reshaped_params = tf.reshape(params,
+                                   [params.shape[0]] +
+                                   [1] * (previous_state.shape.rank - 1))
+      broadcasted_params = tf.broadcast_to(reshaped_params,
+                                           previous_state.shape)
+      reshaped_dist = independent.Independent(
+          normal.Normal(previous_state + broadcasted_params + 1, 0.1),
```

Comment:
`params` is has shape `[num_outer_particles]` and `previous_state` has shape `[num_outer_particles, num_inner_particles, 2]`, so here it should be sufficient to just do:
```
previous_state + params[..., tf.newaxis, tf.newaxis] + 1
```